app/FrontModule/templates/Talkba.author.latte

```diff
@@ -22,19 +22,19 @@
     </div>
 
     {if isset($author->username)}
-        {include $themeDir."/inc_komunita.latte", is_wide => true, author_list => [$author]}
+        {include $themeDir."/inc_zajednica.latte", is_wide => true, author_list => [$author]}
     {else}
         <div class="pull-right">
-                <i>Pozn.: Profil OSMCZ nebyl založen, možnost je dole na stránce <a href="/komunita">komunita</a>
-                        <br>Ikona vlevo je ze služby gravatar.com pro příslušnou emailovou adresu.
+                <i>Napomena: OSMBiH profil nije osnovan, mogućnost osnivanja je dole na stranici <a href="/zajednica">zajednica</a>
+                        <br>Slika lijevo je sa usluge gravatar.com za predmetnu email adresu.
```

Comment:
ne kontam ni prevod ni original

Reply:
Ovo je kad se neko upisuje kao subscriber na mailing list, pa da ne bi mislio da mu se u isto vrijeme stvara i OSM nalog. Kao ako hoćeš i to, ima dole na stranici.

Reply:
o pardon, mislila sam na drugi red- to ne kontam